src/known-organizations/index.js

```diff
@@ -0,0 +1,16 @@
+import aragonGovernanceLogo from './images/aragon-governance.svg'
+import aragonOneLogo from './images/aragon-one.svg'
+
+export const KnownOrganizations = new Map([
+  // governance.aragonproject.eth
+  [
+    '0x2de83b50af29678774d5abc4a7cb2a588762f28c',
+    { name: 'Aragon Governance', image: aragonGovernanceLogo },
+  ],
+
+  // a1.aragonid.eth
+  [
```

Comment:
Can we add Melon as a placeholder pointing it to this DAO for now `0x8A83D4bCE45b4C4F751f76cf565953D1E4A3BF0a`?